messages/en.json

```diff
@@ -12,10 +12,10 @@
   "store/header.print.button": "Print",
   "store/header.thanks": "Thank you for your purchase!",
   "store/items.attachments.subscription": "Subscription",
-  "store/items.attachments.subscription.frequency.day": "Every {frequencyNumber, plural, one {# day} other {# days}}",
-  "store/items.attachments.subscription.frequency.month": "Every {frequencyNumber, plural, one {# month} other {# months}}",
-  "store/items.attachments.subscription.frequency.week": "Every {frequencyNumber, plural, one {# week} other {# weeks}}",
-  "store/items.attachments.subscription.frequency.year": "Every {frequencyNumber, plural, one {# year} other {# years}}",
+  "store/items.attachments.subscription.frequency.day": "{frequencyNumber, plural, one {Every # day} few {Every # days} many {Every # days} other {Every # days}}",
```

Comment:
why do we need to change the English translation though? The languages that need those features can rewrite the whole message to this format.

Reply:
You can have the file `en.json` as:

```json
"store/items.attachments.subscription.frequency.day": "Every {frequencyNumber, plural, one {# day} other {# days}}",
```

And `ru.json` as:

```json
"store/items.attachments.subscription.frequency.day": "{frequencyNumber, plural, one {Foo # bar} few {Foo # bar} many {Foo # bar} other {Foo # bar}}",
```

Reply:
@dvm1984

Reply:
we use en.json as source for all other languages in Crowdin. We could, alternatively, change the en.json source in Crowdin only and go from there, but this will be reflected if we would ever open a PR here with an English languages update.

Reply:
What I'm saying is: why do we need to change the English translation to a more complex one when English don't need all the pluralization options. 

We can instead just make the non-latin languages use this format when needed. The translator is free to change the whole string if it's needed.

Reply:
ok